docs/misc/prune-example.txt

```diff
@@ -53,16 +57,16 @@ Mo Tu We Th Fr Sa Su  Mo Tu We Th Fr Sa Su  Mo Tu We Th Fr Sa Su
 List view
 ---------
 
---keep-daily 14     --keep-monthly 6
+--keep-daily 14     --keep-monthly 6     --keep-yearly 6
```

Comment:
did you mean --keep-yearly 1 (as in the above example)?

Reply:
Whoops, copypasta error. Fixed.